Fixes errors like

Assembler messages:
Fatal error: can't create .libs/pixman-mips-dspr2-asm.o: No such file or directory

it works with glibc because it uses gcc fixed-headers but thats not right.

We move the include under C block

Signed-off-by: Khem Raj <raj.khem@gmail.com>
Upstream-Status: Pending
Index: pixman-0.32.6/pixman/pixman-private.h
===================================================================
--- pixman-0.32.6.orig/pixman/pixman-private.h
+++ pixman-0.32.6/pixman/pixman-private.h
@@ -1,4 +1,3 @@
-#include <float.h>
 
 #ifndef PIXMAN_PRIVATE_H
 #define PIXMAN_PRIVATE_H
@@ -17,6 +16,8 @@
 
 #ifndef __ASSEMBLER__
 
+#include <float.h>
+
 #ifndef PACKAGE
 #  error config.h must be included before pixman-private.h
 #endif
